Map/Source/SquareMap.cpp

```diff
@@ -38,6 +39,50 @@ char SquareMap::printField(const Field& field)
   }
 }
 
+std::string SquareMap::getMapToPrint(const Position& playerPosition)
+{
+	std::stringstream output;
+	
+	const unsigned int mapSize = _fieldList.size();
+	
+	output << "   ";
+	for(unsigned int ite = 0; ite < mapSize; ++ite)
+	{
+		output << ' ' << (ite + 1) % 10;
+	}
+	output << '\n';
+	
+	output << "  ";
+	for(unsigned int ite = 0; ite < mapSize * 2 + 3; ++ite)
```

Comment:
masz tutaj duzo magic value. Ktoś z boku w ogole nie wie o co chodzi.

Powinno to wygladac tak:
constexpr auto nazwaMówiącOTymCzymJestTrójkaWPowyzszymAlgorytnimie = 3u;

for(unsigned int ite = 0; ite < mapSize * 2 + nazwaMówiącOTymCzymJestTrójkaWPowyzszymAlgorytnimie; ++ite)